omegaconf/_utils.py

```diff
@@ -30,6 +30,49 @@
     attr = None  # type: ignore # pragma: no cover
 
 
+# source: https://yaml.org/type/bool.html
+YAML_BOOL_TYPES = [
+    "y",
+    "Y",
+    "yes",
+    "Yes",
+    "YES",
+    "n",
+    "N",
+    "no",
+    "No",
+    "NO",
+    "true",
+    "True",
+    "TRUE",
+    "false",
+    "False",
+    "FALSE",
+    "on",
+    "On",
+    "ON",
+    "off",
+    "Off",
+    "OFF",
+]
+
+
+class OmegaConfDumper(yaml.Dumper):  # type: ignore
+    pass
+
+
+def isbool(b: str) -> bool:
```

Comment:
call this function is_yaml_bool

Reply:
I think those functions are already defined in decode_primitive, refactor. (use those instead of what's in that function, except for the bool one which is a yaml_bool).